src/stories/VideoBox.stories.js

```diff
@@ -0,0 +1,33 @@
+import VideoBox from '~/components/video-box/VideoBox'
+import videoImg from '~/assets/img/guest-home-page/videoImg.png'
+
+export default {
+  title: 'VideoBox',
+  component: VideoBox,
+  argTypes: {
+    video: {
+      type: 'string',
+      description: 'Variable of the image'
+    }
+  }
+}
+
+const Template = (args) => <VideoBox {...args} />
+
+export const Desktop = Template.bind({})
+
+Desktop.args = {
```

Comment:
leave only one state because props are the same everywhere

Reply:
Fixed.